MODERATION_POLICY.md

```diff
@@ -0,0 +1,297 @@
+# Moderation policy
+
+If you are not a member of the webpack GitHub Organization and wish to submit a
+moderation request, please see [Requesting Moderation][]
+
+* [Applicability][]
+* [Terms][]
+* [Grounds for Moderation][]
+* [Requesting Moderation][]
+* [Consideration of Intent][]
+* [Guidelines and Requirements][]
+  * [Collaborator Posts][]
+  * [Non-Collaborator Posts][]
+  * [Temporary Interaction Limits][]
+  * [Temporary and Indefinite Blocks][]
+* [Privacy of the webpack/moderation Repository][]
+* [Escalation of Issues][]
+* [Modifications to This Policy][]
+
+## Applicability
+
+This policy applies to all repositories under the webpack
+GitHub Organization and all webpack Working Groups. This policy also
+applies to the [webpack Slack Community](https://webpack.slack.com),
+supported by the Admin team of the Slack organization.
+
+## Terms
+
+* *Collaborator* refers to any individual with configured triage role or higher
+  in any webpack GitHub organization repository. See
+  [GitHub's Repository roles documentation][] for more information.
+* *TSC* refers to the [webpack Technical Steering Committee][].
+* *Post* refers to the content and titles of any issue, pull request, comment,
+  discussion, or wiki page.
+* *Moderate* means to modify, lock, or delete one or more Posts to correct or
+  address Code of Conduct violations.
+* *Remove* refers to the act of removing the configured write (commit)
+  permissions for an individual Collaborator's GitHub account from *all*
+  webpack GitHub Organization repositories as well as removing the account from
+  the webpack GitHub Organization membership.
+* *Block* refers to the act of prohibiting an individual GitHub account from any
+  further participation in the webpack GitHub Organization. A block may be
+  *temporary* or *indefinite*.
+  * This Moderation Policy applies only to blocking from the organization.
+    Individuals may choose to
+    [block other individuals from their personal GitHub accounts][]. This policy
+    does not restrict blocking from personal GitHub accounts.
+* *Requester* refers to an individual requesting Moderation on a Post.
+
+## Grounds for Moderation
+
+Any Post in violation of the webpack [Code of Conduct][] is subject
+to Moderation.
+
+The TSC is responsible for deciding what constitutes inappropriate
+behavior that may be subject to Moderation.
+
+## Requesting Moderation
+
+Anyone may request Moderation of a Post. Requesting Moderation of a Post can be
+accomplished in one of four ways:
+
+* Via the [report@webpack.js.org][] email address,
```

Comment:
@alexander-akait you got access to this email account now?

Reply:
No, WIP on this

Reply:
🙏 -- this email doesn't exist, but needs to be created.